client.js

```diff
@@ -72,12 +72,85 @@
 		readyCallbacks = {},
 		communicators = {};
 
+	var _getOption = function (key, defaultValue) {
```

Comment:
I like giving the ability to set options as the second argument to the constructor. :+1:

Reply:
:+1: